CMake/VeloxUtils.cmake

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+include_guard(GLOBAL)
+
+function(velox_add_dependencies TARGET)
+  if(VELOX_MONO_LIBRARY AND "${TARGET}" MATCHES "^velox_*")
+
```

Comment:
This looks odd, I would add a comment confirming it is not doing anything or negate the conditional to skip the empty `if` clause.